src/webgpu/api/operation/memory_sync/buffer/multiple_buffers.spec.ts

```diff
@@ -178,3 +178,51 @@ g.test('ww')
     The second write should overwrite the contents of the first.`
   )
   .unimplemented();
+
+g.test('two_dispatches_in_the_same_compute_pass')
+  .desc(
+    `
+  Test write-after-write operations in the same compute pass on multiple buffers. The first write
+  will write 1 into storage buffers. The second write will write 2 into the same buffer in the same
+  pass. Expected data in each buffer is 2.
+  `
+  )
+  .fn(async t => {
+    const buffers: GPUBuffer[] = [];
+
+    const kBufferCount = 4;
+
+    for (let i = 0; i < kBufferCount; ++i) {
+      const buffer = await t.createBufferWithValue(0);
+      buffers.push(buffer);
+    }
+
+    const passes: GPUComputePassEncoder[] = [];
+    const encoders: GPUCommandEncoder[] = [];
+
+    for (let i = 0; i < kBufferCount; ++i) {
```

Comment:
I'm not fully sure if we need to test this on multiple buffers, but I refer to this test from singlie_buffer.spec.ts, then I make multiple buffers with 0 initial value. And, I check if the second writing writes 2 into the same buffers. I wonder if this test scenario makes sense. wdyt?

Reply:
Ah, I understand what's your point on this test. Let me apply this to the PR.

Reply:
BTW, `2 * b` is added to distinguish each pair of dispatches, right?

Reply:
Yes, it's so every dispatch will write a different value to the buffer.